README.md

```diff
@@ -26,28 +26,34 @@ $ sudo docker pull ghcr.io/ledgerhq/ledger-app-builder/ledger-app-builder-legacy
 
 ## Compile your app in the container
 
-In the source folder of your application, for Nano S:
+The `BOLOS_SDK` variable is used to specify the target SDK, allowing to compile the application for each Ledger device. 
 
+In the source folder of your application, you can compile with the following commands:
+
+For Nano S
```

Comment:
We can add bullet points for readability (for each device) @tdejoigny-ledger 

`* For Nano S`
or a subsection
`### For Nano S`